src/applications/static-pages/ezr-tera-alert/components/App/index.jsx

```diff
@@ -14,22 +14,16 @@ export const App = ({ isEzrEnabled }) => {
             On <strong>March 5, 2024</strong>, we expanded health care to
             millions of Veterans.
           </p>
-          <p className="vads-u-margin-top--0">
+          <a href="https://www.va.gov/resources/the-pact-act-and-your-va-benefits/">
```

Comment:
This should be converted to a `va-link`. 

`<va-link href="/resources/the-pact-act-and-your-va-benefits/" text=" Learn more about the PACT Act and VA health care and benefits" />`

Reply:
You also have an action link in the `else` alert. Do you need to replace both action links?

Reply:
Good catch Matt! And I thought that there had to be a VA version of the anchor tag thanks for pointing that out!